packages/sn-redux/src/Actions.ts

```diff
@@ -584,3 +585,52 @@ export const getSchema = () => ({
       method: 'GET',
     }),
 })
+
+/**
+ * Action creator for sharing a content
+ * @param options object with the sharing options(identity, level, mode, etc).
+ */
+export const share = (options: SharingOptions) => ({
+  type: 'SHARE',
+  payload: (repository: Repository) =>
+    repository.share({
+      content: options.content,
+      identity: options.identity,
+      sharingLevel: options.sharingLevel,
+      sharingMode: options.sharingMode,
+      sendNotification: options.sendNotification || true,
```

Comment:
This is going to be true even if you don't want to send notification. I suggest to use the new nullish coalescing operator (??) in situations like this.
```suggestion
      sendNotification: options.sendNotification ?? true,
```